### DISCUSSION TITLE
1.22.1

### DISCUSSION THREAD

**Discussion by acaonweb** [Koenkk/zigbee2mqtt#10188] [General]
Ciao, what about 1.22.1? It's needed to upgrade firmware for a zzh! CC2652R Multiprotocol RF Stick and CC2531 stick? Thanx or i can update only zigbee2mqtt? and what about CC2531 used ad router? let me know :):)

Fabrizio

**Comment by Koenkk** [comment]
Firmware upgrade is not needed.
